### PR TITLE
Add Mii maker

### PR DESCRIPTION
**Pull request by nabilhoche**
Adds Mii maker.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/mii-maker/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Mii maker"
```
Comment (Leo626) [inline games/mii-maker/game.dat]: `Mii Maker`

File: games/mii-maker/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+description = "mii maker It is an application that serves to create mii"
```
Comment (Leo626) [inline games/mii-maker/game.dat]: `Mii Maker is an application that serves to create Miis.`

File: games/mii-maker/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+needs_system_files = true
+needs_shared_font = false
+
+[[releases]]
+title = "1234567891234567"
```
Comment (Leo626) [inline games/mii-maker/game.dat]: This is not a title ID.

File: games/mii-maker/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+region = "jpn"
+release_date = "2011-03-25"
```
Comment (Leo626) [inline games/mii-maker/game.dat]: Should be `2011-02-26` as the Japanese 3DS released on February 26, 2011.

File: games/mii-maker/game.dat [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+[[testcases]]
+title = "000400000009F200"
```
Comment (Leo626) [inline games/mii-maker/game.dat]: This is the wrong title ID (see [Title list](https://www.3dbrew.org/wiki/Title_list#00040010_-_System_Applications)).

File: games/mii-maker/game.dat [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+compatibility = "0"
+date = "2011-03-25"
```
Comment (Leo626) [inline games/mii-maker/game.dat]: Not the current test date.

**Comment by TheKoopaKingdom** [comment]
@nabilhoche please fix the issues mentioned above, or this thread will be closed.

**Pull request closed without merging**